src/VisualStudio/Core/Def/Implementation/LanguageService/AbstractLanguageService`2.IVsImmediateStatementCompletion2.cs

```diff
@@ -25,9 +24,22 @@ int IVsImmediateStatementCompletion2.EnableStatementCompletion(int enable, int s
         {
             if (filters.TryGetValue(textView, out var filter))
             {
+                // Disables/enables the old Roslyn completion.
```

Comment:
How? Didn't we conclude this didn't work anymore?

Reply:
Thank you! Double checked by disabling this line. It does not change anything. Removing it.

---
In reply to: [270162682](https://github.com/dotnet/roslyn/pull/32631#discussion_r270162682) [](ancestors = 270162682)

Reply:
You _may_ want to leave it for now just in case we're wrong, but the comment most definitely isn't correct.